lxl-web/src/lib/components/Search.svelte

```diff
@@ -14,8 +14,10 @@
 		: $page.url.searchParams.get('_i')?.trim();
 
 	let params = getSortedSearchParams(getDefaultSearchParams($page.url.searchParams));
-	params.set('_offset', '0'); // Always reset offset on new search
-	params.delete('_i'); // reset '_i' param on new search
+	// Always reset these params on new search
+	params.set('_offset', '0');
+	params.delete('_i');
+	params.delete('_o');
```

Comment:
```suggestion
	params.delete('_o');
	params.delete('_p');
```

https://github.com/libris/lxlviewer/pull/1039

Reply:
Ah! One step ahead 👀